frontend/src/metabase/redux/user.js

```diff
@@ -35,6 +36,18 @@ export const currentUser = handleActions(
     [CLOSE_QB_NEWB_MODAL]: {
       next: (state, { payload }) => ({ ...state, is_qbnewb: false }),
     },
+    [Users.actionTypes.UPDATE]: {
```

Comment:
Updating user info didn't update the current user. This change is necessary for reflecting the name change under user avatar (new UI) in user profile page.
![image](https://user-images.githubusercontent.com/1937582/173746219-e5228cdb-cac6-4a89-9514-89241bd6f25f.png)

Reply:
Nice catch!